src/pybind/mgr/dashboard/frontend/src/app/shared/components/text-label-list/text-label-list.component.html

```diff
@@ -0,0 +1,24 @@
+<div>
```

Comment:
do we need this div? it looks empty and can be avoided if i am not wrong?

Reply:
We don't need it, you are right, updated